apps/site/lib/site_web/controllers/trip_plan_controller.ex

```diff
@@ -66,53 +66,70 @@ defmodule SiteWeb.TripPlanController do
     redirect(conn, to: trip_plan_path(conn, :index, Map.delete(params, "address")))
   end
 
+  def to(conn, %{
+        "address" => <<latitude::bytes-size(9)>> <> "," <> <<longitude::bytes-size(10)>> = address
```

Comment:
can it always be assumed that the lat/long portions of the address prop will be these lengths?

Reply:
Good question! Definitely isn't a guarantee despite a spot check of many different stops (this is hit when using "get directions to this station" from a stop page, which usually goes to that stop's lat/lng) - I should probably just check if these are floats (of any length) instead

Reply:
Okay, adjusted my approach to this one